hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -367,4 +384,42 @@ public int hashCode() {
             return key == null ? 0 : key.hashCode();
         }
     }
+
+    /**
+     * Defines the parameters used when evicting entries from the cache.
+     */
+    public interface EvictionConfig {
+        /**
+         * @return the duration for which an item should live in the cache
+         */
+        Duration getTimeToLive();
```

Comment:
The hib53 module is Java8, so can use the Duration type to make this slightly nicer